trafficgen/reporter.py

```diff
@@ -103,12 +104,28 @@ def generate_report(input_json):
 def main():
     process_options()
 
+    file_processed = False
+
     try:
         input_fp = open(t_global.args.input, 'r')
         input_json = json.load(input_fp)
         input_fp.close()
+        file_processed = True
 
     except:
+        pass
+
+    if not file_processed:
+        try:
+             input_fp = lzma.open(t_global.args.input, 'r')
+             input_json = json.load(input_fp)
+             input_fp.close()
+             file_processed = True
+
+        except:
+             pass
```

Comment:
I'm attempting to transparently handle failures due to the file being compressed or not.

Reply:
Also, not saying this is right, but since it has never reported the exact exception it is actually going to give the exact same level of error message that it did before.